src/Nethermind/Nethermind.Evm.Test/WithdrawalRequestsProcessorTests.cs

```diff
@@ -49,7 +49,7 @@ public void Setup()
         _stateProvider.Commit(_specProvider.GenesisSpec);
         _stateProvider.CommitTree(0);
 
-        _codeInfoRepository = new CodeInfoRepository();
+        _codeInfoRepository = new CodeInfoRepository(1);
```

Comment:
Let's avoid magic numbers (`MainnetSpecProvider.ChainId == 1`)

```suggestion
        _codeInfoRepository = new CodeInfoRepository(_specProvider.ChainId);
```

Reply:
wrong PR